pallets/ethereum-transactions/src/ethereum_transaction.rs

```diff
@@ -325,6 +324,19 @@ impl EthAbiHelper {
         return ethabi::encode(&call.call_values)
     }
 
+    pub fn generate_confirmation_data_for_compacted_calls(
```

Comment:
I can't see how the function name matches what it does. There is no confirmation logic here, as far as I can see, this function is abi encoding call data parameters.